test/TestPercentageMath.sol

```diff
@@ -43,27 +43,38 @@ contract TestPercentageMath is Test {
     /// TESTS ///
 
     function testPercentMul(uint256 x, uint256 y) public {
-        unchecked {
-            if (y > 0 && x > MAX_UINT256_MINUS_HALF_PERCENTAGE / y) {
-                vm.expectRevert();
-                PercentageMath.percentMul(x, y);
-            }
-        }
+        vm.assume(y == 0 || (y > 0 && x <= MAX_UINT256_MINUS_HALF_PERCENTAGE / y));
```

Comment:
```suggestion
        vm.assume(y == 0 || x <= MAX_UINT256_MINUS_HALF_PERCENTAGE / y);
```

Reply:
Not sure that this is possible (div by 0)

Reply:
oh no you are right, if the first assertion is true, it doesn't evaluate the second one in an OR.